src/rtp.cpp

```diff
@@ -169,26 +169,27 @@ void RtcpReportBlock::preparePacket(SSRC in_ssrc, [[maybe_unused]] unsigned int
 
 void RtcpReportBlock::setSSRC(SSRC in_ssrc) { _ssrc = htonl(in_ssrc); }
 
-void RtcpReportBlock::setPacketsLost([[maybe_unused]] unsigned int packetsLost,
-                                     [[maybe_unused]] unsigned int totalPackets) {
-	// TODO Implement loss percentages.
-	_fractionLostAndPacketsLost = 0;
+void RtcpReportBlock::setPacketsLost(uint8_t packetsLost,
+                                     unsigned int totalPackets) {
+	_fractionLostAndPacketsLost = ( (uint32_t)packetsLost << 24) && htonl(totalPackets);
 }
 
-unsigned int RtcpReportBlock::getLossPercentage() const {
-	// TODO Implement loss percentages.
-	return 0;
+uint8_t RtcpReportBlock::getLossPercentage() const {
+	// Loss percentage is expressed as 8-bit fixed point number
+	// In order to get actual percentage divide the result by 256
+	return _fractionLostAndPacketsLost & 0xFF;
```

Comment:
As the function just returns the fraction lost as fixed point, I think you should rename it `getFractionLost` otherwise it it can be quite confusing.